cdap-proto/src/main/java/co/cask/cdap/proto/ProtoTrigger.java

```diff
@@ -35,10 +37,37 @@
    * Represents all known trigger types in REST requests/responses.
    */
   public enum Type {
-    TIME,
-    PARTITION,
-    STREAM_SIZE,
-    PROGRAM_STATUS
+    TIME("time"),
```

Comment:
So would the "category name" for these types just be used for receiving requests, or can this also be used to define how types are sent back?

I was looking at the TriggerCodec and saw that when its deserialize method is called, the type is defined like:

```
{"cronExpression":"0/15 * * * * ?","type":"TIME"}
```